docs/commands/build-docs.md

```diff
@@ -0,0 +1,113 @@
+# `build-docs`
+
+## Introduction
+
+The `build-docs` command builds Redoc into a zero-dependency HTML file.
+
+## Usage
+
+```bash
+redocly build-docs <api>
+redocly build-docs <api> --output=custom.html
+redocly build-docs <api> --features.openapi.theme.colors.primary.main=orange
+redocly build-docs <api> -t custom.hbs
+redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page meta description"
+```
+
+
+## Options
+
+Option | Type | Description
+-- | -- | --
+api | string | Path to the API definition filename or alias that you want to generate the build for. Refer to the [API examples](#api-examples) for more information.
+--output | string | Sets the path and name of the output file. The default value is `redoc-static.html`.
+--title | string | Sets the page title.
+--disableGoogleFont | boolean | Disables Google fonts. The default value is `false`.
+--cdn | boolean | Uses the CDN pointing to the latest version of Redoc. If not enabled, the Redoc version is selected from Redocly CLI's associated dependency (and to update Redoc requires updating the CLI and building the docs again). The default value is `false`.
+--template | string | Uses custom [Handlebars](https://handlebarsjs.com/) templates to render your OpenAPI definition.
+--templateOptions | string | Adds template options you want to pass to your custom Handlebars template. To add options, use dot notation.
+--features.openapi | string | Customizes your output using [Redoc functionality options](https://redocly.com/docs/api-reference-docs/configuration/functionality) or [Redoc theming options](https://redocly.com/docs/api-reference-docs/configuration/theming).
```

Comment:
⚠️ Inconsistency alert: the other options use camelCase. This one uses a dot-separator. This one is consistent with the naming in the configuration file. Should anything change? Should the configuration file change to `featuresOpenapi`? Thoughts? cc @RomanHotsiy

Reply:
hmmm... Good question... Maybe `openapiFeatures`?

Reply:
Shouldn't it be consistent with `config.yaml` file? 
That was previously discussed [here](https://github.com/Redocly/redocly-cli/pull/871#issuecomment-1257571473)

Reply:
We talked and suggested `oasFeatures`, as well as updating the config file format to match. And eventually, `graphqlFeatures`, etc...

Reply:
added for `build-docs` command in [PR](https://github.com/Redocly/redocly-cli/pull/892)

Reply:
@RomanHotsiy and I talked more and maybe it will be `features`?

Reply:
And in this case, it may be a nested object, and therefore features.openapi.hideTryIt would map to:

```
features:
  openapi:
     hideTryIt: true
```

Reply:
We looked at this in more depth, and believe it should be:

```yaml
theme:
  openapi:
    hideTryIt: true
```